packages/transactions/src/OutboundShipment/DetailView/tabs/GeneralTab.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  ColumnDefinition,
+  Item,
+  RemoteDataTable,
+  useColumns,
+  useDataTableApi,
+  useQueryParams,
+  useSortedData,
+} from '@openmsupply-client/common';
+import React, { FC } from 'react';
+
+interface GeneralTabProps<T> {
+  data: T[];
+}
+
+const defaultColumns: ColumnDefinition<Item>[] = [
+  {
+    label: 'label.code',
+    key: 'code',
+    width: 100,
+    minWidth: 100,
+    maxWidth: 100,
+    align: 'left',
+  },
+  {
+    label: 'label.name',
+    key: 'name',
+    width: 100,
+    minWidth: 100,
+    maxWidth: 100,
+    align: 'left',
+  },
+  {
+    label: 'label.packSize',
+    key: 'packSize',
+    width: 100,
+    minWidth: 100,
+    maxWidth: 100,
+    align: 'left',
```

Comment:
:nit I'd right align the numeric columns and make them a little less wide than the name column